pyproject.toml

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 [build-system]
-requires = ["rich", "setuptools", "setuptools-scm"]
+requires = ["rich", "setuptools>80", "setuptools-scm"]
```

Comment:
>80 or >=80?

Reply:
The lowest "80" version is `80.8.0` AFAIK so I didn't think it mattered.

Reply:
I think it's instructive for a future reader to know exactly what version introduced what we need. >80 could mean 81, or 80.0.1

Reply:
In any case, not a huge deal, I won't hold up the PR over this